Source/DafnyRuntime/DafnyRuntimeGo/System_/System_.go

```diff
@@ -4,47 +4,47 @@
 package _System
 
 import (
-  _dafny "dafny"
-  os "os"
+	_dafny "dafny"
```

Comment:
I take it these changes are because `goimports` applies auto-formatting as well? 

No issues with that and in fact that's kind of a nice side-effect of this change!